tests/index.test.ts

```diff
@@ -275,6 +275,54 @@ describe('Result.Err', () => {
 })
 
 describe('Utils', () => {
+  describe('fromThrowable', () => {
+    it('Creates a function that returns an OK result when the inner function does not throw', () => {
+      const hello = (): string => 'hello'
+      const safeHello = fromThrowable(hello)
+
+      const result = hello()
+      const safeResult = safeHello()
+
+      expect(safeResult.isOk()).toBe(true)
+      expect(safeResult.isErr()).toBe(false)
```

Comment:
There's already a test case that ensures that if `isOk` is true then `isErr` is false, and vice versa. No need to assert both here.

Reply:
Will remove